cmd/tnf/claim/compare/utils/utils.go

```diff
@@ -0,0 +1,52 @@
+package utils
```

Comment:
This directory seems unused

Reply:
That was a leftover from previous local tests. Removed in https://github.com/test-network-function/cnf-certification-test/pull/1386/commits/ea85a69712d057644d1de6d059567a236e8b0d7f